CMakeLists.txt

```diff
@@ -254,6 +255,7 @@ us_cache_var(US_BUILD_TESTING OFF BOOL "Build tests")
 us_cache_var(US_BUILD_EXAMPLES OFF BOOL "Build example projects")
 us_cache_var(US_USE_SYSTEM_GTEST OFF BOOL "Build using an external GTest installation" ADVANCED)
 us_cache_var(US_USE_SYSTEM_BOOST OFF BOOL "Build using an external Boost installation" ADVANCED)
+us_cache_var(US_USE_LTO ON BOOL "Use link time optimization if supported" ADVANCED)
```

Comment:
this comment refers to "link time optimization" when most of the other code refers to IPO or "interprocedural optimization". I recommend using consistent terms throughout the code.

Reply:
right you are